linux_sql/scripts/host_usage.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/sh
+
+#Setup and validate arguments (again, don't copy comments)
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+
+#Check # of args
+if [ $# -ne  5 ]; then
+  echo 'Not enough information'
```

Comment:
The Message could be more descriptive. What exactly is missing? 
Perhaps something like:
```
Wrong number of arguments. 
```